public/funding.json

```diff
@@ -21,8 +21,8 @@
         "wellKnown": "https://www.openstreetmap.org/.well-known/funding-manifest-urls"
       },
       "repositoryUrl": {
-        "url": "https://planet.openstreetmap.org",
-        "wellKnown": "https://planet.openstreetmap.org/.well-known/funding-manifest-urls"
+        "url": "https://github.com/openstreetmap/",
+        "wellKnown": "https://github.com/openstreetmap/chef/tree/master/cookbooks/web/files/default/static/.well-known/funding-manifest-urls"
```

Comment:
We should be hosting this file, not relying on github. Additionally, it's unlikely that this URL will meet the format required, as it's not the URL of the file, it's a URL for a web page showing the github interface.